### PR TITLE
Code highlight

### PR DESCRIPTION
**Pull request by JakeSidSmith**
Relies on #6

### CODE REVIEW

File: examples/src/js/components/code-block.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import hljs from '../../../../node_modules/highlightjs/highlight.pack';
+
+import { constructClassName, removeIndentation } from '../utils';
+
+export default class CodeBlock extends React.Component {
+  formatCode(code) {
+    return hljs.highlight('javascript', removeIndentation(code)).value;
+  }
+
+  constructClassName() {
```
Comment (RealOrangeOne) [inline examples/src/js/components/code-block.js]: Doesnt seem right having the functions named the same...

File: package.json [hunk 2]
```diff
@@ -4,9 +4,10 @@
   "description": "A simple flux architecture that combines actions & reducers and maps your store's state directly to your components",
   "main": "lib/index.js",
   "scripts": {
+    "copy-hljs": "mkdir -p examples/build/css && cp node_modules/highlightjs/styles/github-gist.css examples/build/css/highlightjs.css",
```
Comment (RealOrangeOne) [inline package.json]: Surely this _hack_ can be done by requiring `hljs`?

Comment (JakeSidSmith) [inline package.json]: I do require `hljs`, this *hack* copies the CSS into the build folder.

Comment (RealOrangeOne) [inline package.json]: Aaahh ok. Maybe you need a CSS pre-processor ;)

Comment (JakeSidSmith) [inline package.json]: I assume it's already processed. 😛

Comment (ekonstantinidis) [inline package.json]: Looks good! :sparkles:

**Pull request merged**